orchestra/models.py

```diff
@@ -461,6 +461,43 @@ def __str__(self):
             str(self.task), self.assignment_counter, str(self.worker))
 
 
+class Iteration(models.Model):
```

Comment:
If TaskAssignment attaches to Task, should this be called TaskAssignmentIteration or TaskIteration or AssignmentIteration?